stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/textinput/StreamMessageInputViewModelBinding.kt

```diff
@@ -0,0 +1,16 @@
+@file:JvmName("StreamMessageInputViewModelBinding")
```

Comment:
We didn't prefix these with `Stream` before, do we want to adopt this new naming style?

Reply:
Added as a topic to our Weekly

Reply:
I think the decision here was to _not_ prefix class names, right?

Reply:
Removed `Stream` prefix for `MessageInputView` and `MessageInputViewModelBinding`. As discussed earlier, adding prefixes to resources in a library makes perfect sense as library users can accidentally override them. This problem does not apply to classes so to be consistent with what we have currently I'm omitting `Stream` prefix.